modules/ossm-members.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v1x/installing-ossm.adoc
+// * service_mesh/v2x/installing-ossm.adoc
+
+[id="ossm-members_{context}"]
+= Creating the {ProductName} members
+
+`ServiceMeshMember` resources can be created by service mesh users who do not have privileges to add members to the `ServiceMeshMemberRoll` directly. While project administrators are automatically given permission to create the `ServiceMeshMember` resource in their project, they cannot point it to any `ServiceMeshControlPlane` until the service mesh administrator explicitly grants access to the service mesh. Administrators can grant users permissions to access the mesh by granting them the `mesh-user` user role. In this example, `istio-system` is the control plane namespace.
```

Comment:
I'd start with the value proposition here: 

> `ServiceMeshMember` resources provide a way for service mesh administrators to delegate permissions to add projects to a service mesh, even when the respective users don't have direct access to the service mesh project or member roll`

Reply:
@dgn Thanks! I added that.

Is `apiVersion: rbac.authorization.k8s.io/v1` the correct apiVersion for the RoleBinding?